<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playWidget</class>
 <widget class="QWidget" name="playWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>playWidget</string>
  </property>
  <widget class="QPushButton" name="beginButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>650</y>
     <width>80</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;微软雅黑&quot;;
background-color:rgb(0, 170, 255);
</string>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_11">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_24">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>90</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_31">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>160</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_32">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>130</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_33">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>90</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_34">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>120</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_41">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>160</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_42">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>90</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_43">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>150</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_12">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>180</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_13">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>110</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_14">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>120</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_21">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>170</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_22">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>150</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_23">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>90</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_44">
   <property name="geometry">
    <rect>
     <x>-200</x>
     <y>190</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
